src/main/java/com/github/netelli/JDBCStuding.java

```diff
@@ -1,10 +1,13 @@
 package com.github.netelli;
 
 import com.github.netelli.model.Product;
+import org.apache.log4j.Logger;
 
 import java.util.List;
 
 public class JDBCStuding {
+    private final static Logger logger = Logger.getLogger(JDBCStuding.class);
+
     public static void main(String[] args) throws Exception {
         try (ProductsDAO productsDAO = new ProductsDAO("jdbc:h2:mem:test")) {
```

Comment:
add some information about start and end of application work, add information about actions that you do (inside DAO in this case), because currently you just print objects and it doesn't give an information to user what's going on and what this data actually means.
Purpose of logging - is to provide explanation about what is going on in program, what is progress of processing, etc..